src/constants/locConstants.ts

```diff
@@ -643,10 +643,15 @@ export let executionPlan = l10n.t("Execution Plan");
 export let scriptCopiedToClipboard = l10n.t("Script copied to clipboard");
 export let copied = l10n.t("Copied");
 
-export let enableRichExperiencesPrompt = l10n.t(
-    "The MSSQL extension has some new GUI-based experiences! Would you like to enable them?",
-);
-export let enableRichExperiences = l10n.t("Enable Rich Experiences");
+export function enableRichExperiencesPrompt(learnMoreUrl: string) {
```

Comment:
NIT: Maybe rename the variable to `enableModernFeatures` to match the prompt.

Reply:
I like that better.  "rich" always felt a bit buzwordy to me

Reply:
I think I'm going to not change it because it's not user-facing (outside of manually editing the config json), and is kind of a sprawling (albeit simple find-replace) change.